recipe_scrapers/__init__.py

```diff
@@ -128,6 +129,7 @@
     Eatsmarter.host(domain="de"): Eatsmarter,
     EatWhatTonight.host(): EatWhatTonight,
     Epicurious.host(): Epicurious,
+    FarmhouseDelivery.host(): FarmhouseDelivery,
```

Comment:
If there are multiple TLDs that are valid can you associate them all with this scraper? See the `Eatsmarter` entry above for an example.

Reply:
There should just be the single domain.

Reply:
Oh, I made that comment because it looked like you're testing for a `.co.uk` domain in the test files. Line 23 on both.

Reply:
I think I thought was standard practice since I used hello fresh as my first example to learn from.

Reply:
I made the same mistake with my sunbasket parser as well.  I will go back and fix that one as well as the others that don't have the alternate domain.